src/test/java/space/obminyashka/items_exchange/end2end/UserFlowTest.java

```diff
@@ -71,14 +73,14 @@ void getUserDto_shouldReturnUserDtoIfExists() throws Exception {
     @Test
     @WithMockUser(username = "admin")
     @DataSet("database_init.yml")
-    @ExpectedDataSet(value = "user/update.yml", ignoreCols = {"last_online_time", "updated"})
-    void updateUserInfo_shouldUpdateUserData() throws Exception {
-        getResultActions(HttpMethod.PUT, "/user/info",
-                createUserDtoForUpdatingWithChangedEmailAndFNameApAndLNameMinusWithoutPhones(), status().isAccepted())
-                .andDo(print())
-                .andExpect(jsonPath("$.email").value(NEW_VALID_EMAIL))
-                .andExpect(jsonPath("$.firstName").value(NEW_VALID_NAME_WITH_APOSTROPHE))
-                .andExpect(jsonPath("$.lastName").value(NEW_VALID_NAME_WITH_HYPHEN_MINUS));
+    @ExpectedDataSet(value = "user/update.yml", ignoreCols = {"last_online_time", "email", "updated"})
+    void updateUserInfo_Success_shouldReturnHttpStatusAccept() throws Exception {
+        UserWithoutChildrenDto dto = createValidUserWithoutChildrenDtoForUpdating();
+
+        getResultActions(HttpMethod.PUT, USER_INFO, dto, status().isAccepted())
+                .andExpect(jsonPath("$.firstName").value(NEW_VALID_NAME))
+                .andExpect(jsonPath("$.lastName").value(NEW_VALID_NAME_WITH_HYPHEN_MINUS))
+                .andExpect(jsonPath("$.phones", hasSize(NEW_VALID_PHONES.size())));
```

Comment:
Size could be unchanged if user had 2 phones and sent new 2 phones instead. You should validate the content.